ci/build/artefact-signing-key-sealed-secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: artefact-signing-key
+  namespace: verify-eidas-proxy-node-build
+spec:
+  template:
+    metadata:
+      name: artefact-signing-key
+      namespace: verify-eidas-proxy-node-build
```

Comment:
```suggestion
      namespace: verify-proxy-node-build
```

Reply:
I've removed this whole `template` block as I'm not convinced it's needed/adding anything useful.